doc/contributor-how-to/how-to-dev-plugin.md

```diff
@@ -0,0 +1,64 @@
+# How to install QGreenland Custom as a developer
+
+QGreenland Custom is a QGIS plugin for downloading a custom set of data, including data which 
+is not part of the QGreenland Core zip package. This how-to guide walks through how to install the plugin as a developer.
+
+First, clone the QGreenland Custom repo from Faunalia:
+
+```
+git clone https://github.com/faunalia/qgreenland-plugin.git
+```
+
+Use the QGIS GUI to locate the QGreenland Custom directory by going to
+Settings -> User Profiles -> Open Active Profile Folder.
+Then, navigate to the /python/plugins directory, PWD and copy the full filpath. 
```

Comment:
This is fairly unclear because "PWD" on its own it not runnable. Should the user execute the `pwd` or `echo $PWD` command? And then they should copy the output of that command.